test/attributes.js

```diff
@@ -325,4 +325,36 @@ describe('Attributes', function () {
       this.validator.validate({quux: 1, foo: 1, bar: 1}, {'dependencies': {'quux': ['foo', 'bar']}}).valid.should.be.true;
     });
   });
+
+  describe('format', function () {
+    beforeEach(function () {
+      this.validator = new Validator();
+      this.validator.addFormat('integer-positive', /^[0-9]+$/);
+      this.validator.addFormat('alphanumeric', /^[a-zA-Z0-9]+$/);
+    });
+
+    it('should validate 1 as valid with integer-positive format', function () {
+      return this.validator.validate(1, {'format': 'integer-positive'}, {}, {'propertyPath': 'apath'}).valid.should.be.true;
+    });
+
+    it('should validate 0 as valid with integer-positive format', function () {
+      return this.validator.validateSchema(0, {'format': 'integer-positive'}, {}, {'propertyPath': 'apath'}).valid.should.be.true;
```

Comment:
This test isn't running `validators.format()`, so I don't think it's testing what you think it is